chart/values.yaml

```diff
@@ -37,6 +37,10 @@ debug: false
 # When starting Rancher for the first time, bootstrap the admin as restricted-admin
 restrictedAdmin: false
 
+# Control how the Rancher agents validate TLS connections
+# Valid options: strict, system-store, null, or leave it undefined
+# agentTLSMode: strict
```

Comment:
I would leave the last item ("leave it undefined" off, and instead say "defaults to null, which does x").